src/client/components/app/ActionButtons/index.tsx

```diff
@@ -1,17 +1,17 @@
-import styled from 'styled-components';
+import styled from 'styled-components/macro';
```

Comment:
I really don't think it will change much, this was the implementation @pentcle decided for, I don't think it has for the moment a downside compared to using a plugin

Reply:
The downside is that now we have to be carefull to import from macro. Even in this PR I think ~40% or more of the files were only a change into how we import this, seems like an overhead for a solution that could be really simple

Reply:
Removed in 06be0ae9806878b4de645ae9d11f19afe5ac0c0a

Resolve with @pentcle later

Reply:
@FiboApe lets implement in another PR adding the plugin then. Lets coordinate with @turtlemoji on that one.